specs/zkevm-prover.md

```diff
@@ -9,117 +9,43 @@
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 **Table of Contents**
 
-- [Overview](#overview)
-- [zkEVM Proof](#zkevm-proof)
-- [the ZK Verifier Contract](#the-zk-verifier-contract)
-- [Implementation](#implementation)
-- [RPC](#rpc)
-  - [Protobuf](#protobuf)
-- [Summary of Definitions](#summary-of-definitions)
-  - [Constants](#constants)
+- [zkEVM Prover](#zkevm-prover)
+  - [Overview](#overview)
+  - [ZKEVM Proofs](#zkevm-proofs)
+  - [The ZK Verifier Contract](#the-zk-verifier-contract)
+  - [Prover as an RPC Server](#prover-as-an-rpc-server)
+    - [Protobuf](#protobuf)
+  - [Summary of Definitions](#summary-of-definitions)
+    - [Constants](#constants)
```

Comment:
```suggestion
- [Overview](#overview)
- [ZKEVM Proofs](#zkevm-proofs)
- [The ZK Verifier Contract](#the-zk-verifier-contract)
- [Prover as an RPC Server](#prover-as-an-rpc-server)
  - [Protobuf](#protobuf)
- [Summary of Definitions](#summary-of-definitions)
  - [Constants](#constants)
```

Reply:
applied.

Reply:
Since this toc was generated by doctoc, so it can not be modified. Do you have any idea to change it?

Reply:
Oh I see it. I just suggested it cause the format of toc is different with other docs.